images/ceph/Dockerfile

```diff
@@ -23,7 +23,7 @@ ARG ARCH
 ARG TINI_VERSION
 
 # Run tini as PID 1 and avoid signal handling issues
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${ARCH} /tini
+RUN curl -SL -o /tini https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${ARCH}
 RUN chmod +x /tini
```

Comment:
Make this and the above `RUN` into one RUN statement instead of two `RUN`s/layers.